tests/test_transactions_reader.py

```diff
@@ -127,14 +127,17 @@ def test_transactions_qty_not_integer(testfiles_dir):
                               50.00,
                               0.0,
                               'USD')
+    # Overwrite the qty after creating the object because otherwise the object
```

Comment:
Should this test name be changed, since it doesn't have to be an integer, but rather it has to be a number?

Reply:
Yes good point, I will change it.